Mathlib/CategoryTheory/Limits/Preserves/Shapes/Kernels.lean

```diff
@@ -29,14 +29,45 @@ variable {C : Type u₁} [Category.{v₁} C] [HasZeroMorphisms C]
 
 variable {D : Type u₂} [Category.{v₂} D] [HasZeroMorphisms D]
 
-variable (G : C ⥤ D) [Functor.PreservesZeroMorphisms G]
-
 namespace CategoryTheory.Limits
 
-section Kernels
+namespace KernelFork
+
+variable {X Y : C} {f : X ⟶ Y} (c : KernelFork f) (hc : IsLimit c)
```

Comment:
The only declaration that uses `hc` is `mapIsLimit`, so I would suggest moving it there to avoid confusion.

Reply:
Thanks for the suggestion.